.env

```diff
@@ -1,2 +1 @@
-NEXT_PUBLIC_MEASUREMENTS_URL=https://api.ooni.io
```

Comment:
Why is this no longer set inside of the default production env?

Reply:
This is already included in `.env.production`